src/zope/security/permission.py

```diff
@@ -31,29 +31,36 @@
 
 @implementer(IPermission)
 class Permission(object):
+    """
+    Default implementation of :class:`zope.security.interfaces.IPermission`.
+    """
 
     def __init__(self, id, title="", description=""):
         self.id = id
         self.title = title
         self.description = description
 
 def checkPermission(context, permission_id):
-    """Check whether a given permission exists in the provided context.
+    """
+    Check whether a given permission object exists in the provided
+    context as a utility.
     """
     if permission_id is CheckerPublic:
         return
     if not queryUtility(IPermission, permission_id, context=context):
         raise ValueError("Undefined permission id", permission_id)
 
 def allPermissions(context=None):
-    """Get the ids of all defined permissions
+    """
+    Get the ids of all defined permission object utilities.
```

Comment:
Capitalize IDs to be consistent with other docstrings?

Reply:
Good idea, will do. (It looks like this package as a whole is actually pretty inconsistent with its usage of "id", "ID" and even "Id"; at least after this pass everything will be consistent for a while.)